src/Microsoft.PowerShell.Commands.Utility/commands/utility/trace/MshHostTraceListener.cs

```diff
@@ -62,35 +62,8 @@ internal PSHostTraceListener(PSCmdlet cmdlet)
         [SecurityPermission(SecurityAction.LinkDemand)]
         protected override void Dispose(bool disposing)
         {
-#if CORECLR
             base.Dispose(disposing);
-#else
-            try
-            {
-                if (disposing)
-                {
-                    this.Close();
-                }
-            }
-            finally
-            {
-                base.Dispose(disposing);
-            }
-#endif
-        }
-
-#if !CORECLR
-        /// <summary>
-        /// Closes the dialog and then calls the base class Close
-        /// </summary>
-        [SecurityPermission(SecurityAction.LinkDemand)]
-        public override void Close()
-        {
-            // Call the base class close
-
-            base.Close();
```

Comment:
The `Close` virtual method is back, so maybe we should restore the full clr implementation.